src/widgets/Seperator.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/no-unused-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { GREY } from '../globalStyles/index';
+
+/**
+ * Simple Separator display component rendering a line.
+ *
+ * @prop {Number} width            The width of the seperator
+ * @prop {String} length           The length of the seperator as a percentage string.
+ * @prop {Number} marginBottom     Number of pixels for the bottom margin.
+ * @prop {Number} marginTop        Number of pixels for the bottom margin.
+ * @prop {Number} marginHorizontal Number of pixels for the top & bottom margin.
+ */
+export const Seperator = props => {
```

Comment:
```suggestion
export const Separator = props => {
```
Also the file name. Probably worth search/replace in the repo.

Reply:
fec4ca9ff6ddf924a88a96d48137875124ac2dbd - includes style spelling error fix